Sources/BraintreeCoreSwift/BTConfiguration.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+/// Contains information specific to a merchant's Braintree integration
+@objcMembers public class BTConfiguration: NSObject {
```

Comment:
random q: Do the classes that aren't meant for the merchant to use directly need to have ObjC annotations?

Reply:
For the moment, yes. That way we can use the code in our own un-converted Objc code. But once all our stuff is converted we could remove it in certain classes.